src/System.Web.OData/OData/Query/ODataQueryOptions.cs

```diff
@@ -83,7 +85,22 @@ public ODataQueryOptions(ODataQueryContext context, HttpRequestMessage request)
 
             BuildQueryOptions(queryParameters);
 
-            Validator = ODataQueryValidator.GetODataQueryValidator(context);
+            Validator = ODataQueryValidator.GetODataQueryValidator(context);           
+        }
+
+        /// <summary>
+        /// Gets Column order.
+        /// </summary>
+        public static Dictionary<string, int> ColumnOrder
```

Comment:
I get that static is very convenient here but it would be much better if this were stored as a member somewhere that was model or even entity specific.